dsl/src/main/java/tech/pegasys/peeps/node/genesis/GenesisFile.java

```diff
@@ -22,19 +22,27 @@
 import java.nio.file.Path;
 import java.nio.file.StandardOpenOption;
 
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
 import io.vertx.core.json.DecodeException;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.testcontainers.shaded.org.bouncycastle.util.Arrays;
 
-public class BesuGenesisFile {
+public class GenesisFile {
 
   private static final Logger LOG = LogManager.getLogger();
 
   private final Path genesisFile;
+  private final ObjectMapper objectMapper;
+
+  public GenesisFile(final Path genesisFile) {
 
-  public BesuGenesisFile(final Path genesisFile) {
     this.genesisFile = genesisFile;
+    objectMapper = new ObjectMapper().registerModule(new Jdk8Module());
```

Comment:
for performance reasons this can be part of static initializer.

Reply:
pretty minor improvement, but we can take it!